### DISCUSSION TITLE
iOS support for artwork image

### DISCUSSION THREAD

**Discussion by jeremieoneglancestudio** [mediagrid/capacitor-native-audio#8] [2. Feature Ideas]
Hello,

The artwork url currently work on android for remote url but not iOS. Is it possible to make it work too?

**Comment by jeremieoneglancestudio** [comment]
Hello, any news on this?

**Reply by wsamoht** [reply]
It's on the list to do :) Hopefully in the next week or two. I am currently in the middle of a major upgrade for another application.

For my reference later, a possible solution is https://stackoverflow.com/a/32993613/1129035.

**Comment by wsamoht** [comment]
This has been added for iOS. Can you try out the branch over here - https://github.com/mediagrid/capacitor-native-audio/tree/add-ios-artwork-image-support and let me know if you have any issues?

You should be able to install via

```
npm install https://github.com/mediagrid/capacitor-native-audio#add-ios-artwork-image-support
```

**Reply by wsamoht** [reply]
The branch has been merged into `main`.

You should be able to test by running:

```
npm install https://github.com/mediagrid/capacitor-native-audio#main
```

**Comment by soramister** [comment]
Hello,

I have tested and it works properly. Thank you for your hard work !

**Comment by wsamoht** [comment]
v1.2.0 has been tagged!

https://github.com/mediagrid/capacitor-native-audio/releases/tag/v1.2.0
